scripts/change_react_native_version.js

```diff
@@ -18,7 +18,7 @@ async function run() {
     packageJson.devDependencies['react-native'] = reactNativeVersion;
   } else {
     const data = await fetch(`https://registry.npmjs.org/react-native/${reactNativeVersion}`);
-    const reactVersion = data.peerDependencies.react;
```

Comment:
please let's not start managing these versions with carets. We revisit these every once in a while manually because of the RN upgrades, anyways, so we should be good IMO